src/test/java/org/example/integration/JobRoleControllerIntegrationTest.java

```diff
@@ -0,0 +1,33 @@
+package org.example.integration;
+
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.example.cli.JobRole;
+import org.example.trueApplication;
+import org.example.trueConfiguration;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class JobRoleControllerIntegrationTest {
+    private final String API_URL_JOB_ROLES = "http://localhost:8080/api/job-roles";
```

Comment:
This will have the same issue as before (not working correctly when deployed). Ideally you would set the base URL `http://localhost:8080/api` in this as the value passed tough from an environment variable.